prod/westeurope/internal/api/functions_iopayportal/function_app_slot_staging/terragrunt.hcl

```diff
@@ -101,6 +105,8 @@ inputs = {
     # this app settings is required to solve the issue:
     # https://github.com/terraform-providers/terraform-provider-azurerm/issues/10499
     WEBSITE_CONTENTSHARE = "staging-content"
+
+    IO_PAY_CHALLENGE_RESUME_URL = "https://${dependency.storage_account_iopayportal.outputs.primary_web_host}/response.html?id=idTransaction"
```

Comment:
@uolter is it correct to define https://io-p-cdnendpoint-iopay.azureedge.net/?

Reply:
@infantesimone should be this one `dependency.storage_account_iopay.outputs.primary_web_host`right ?

Reply:
@pasqualespica Thanks!

Reply:
Yes, this is correct: https://io-p-cdnendpoint-iopay.azureedge.net